Testscripts/Linux/SetupRDMA.sh

```diff
@@ -404,17 +444,22 @@ function Main() {
 		# in newer kernels, mad.h is missing from /usr/include/infiniband
 		ls /usr/include/infiniband/ | grep -w mad.h
 		if [[ $? -ne 0 ]]; then
+			LogMsg "Found mad.h file is missing in /usr/include/infiniband/. Copied one from $madh_location"
```

Comment:
here $madh_location is empty

Reply:
fixed in 66ae117079544b049a7a20e2d236a40d7b8e8825